### ISSUE TITLE
Login page redesign

### ISSUE THREAD

**Issue by tingeber** [TheRestartProject/Fixometer#48]
@DLupo to prep visual design for login page 
http://fixometer.zardtech.com/user/login

after: @IronRunes implements

**Comment by DLupo** [comment]
Hi here is a redesign of the login page,

![uploads/4d1cfde3-15a3-4bc7-9eb5-935226407dfa/Fixometer Login screen.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2F4d1cfde3-15a3-4bc7-9eb5-935226407dfa%2FFixometer+Login+screen.png)

**Comment by janetgunter** [comment]
I've given @DLupo feedback and we are working on alternatives.

**Comment by DLupo** [comment]
Hi @janetgunter  here are two alternative designs, 

![uploads/f20c7817-1c63-401a-8f4f-97f906e2b6e0/Fixometer Login-01.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2Ff20c7817-1c63-401a-8f4f-97f906e2b6e0%2FFixometer+Login-01.png)


![uploads/52c47f45-df09-46ec-9ba5-a4ccdfcd7234/Fixometer Login-02.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2F52c47f45-df09-46ec-9ba5-a4ccdfcd7234%2FFixometer+Login-02.png)

**Comment by janetgunter** [comment]
Hi @IronRunes, if we pick a photo-based design, can you have 2-3 different images it chooses from randomly?

Dimitri, I'd like to keep the explanatory blurb bigger (as in the second example)

While I recognise it needs a title, I am really struggling to name it... My idea is that the "Fixometer" is the engine behind this site, but that the site should be called something else @tingeber?

**Comment by tingeber** [comment]
Yes to 2-3 images chosen randomly, @janetgunter please send pics over to @IronRunes 

I agree on title not being "fixometer" - will open new card for that

**Comment by janetgunter** [comment]
So, can we use Dimitri's second design, with "Welcome to our community space" all in large blue Patua typeface, instead of just the "W". And if possible, can we add CO2 emissions prevented? Is there any way to make that text more readable @DLupo?

**Comment by janetgunter** [comment]
I didn't communicate well re text on this page. Can we maintain everything about Dimitri's design 2 (including typeface for the paragraph) with the sole exception of the words "Welcome to our community space" which should be blue Patua?

Also the photos seem to be zoomed and cropped in a way where they are losing quality 

Don't know if the footer is meant to be visible on the same first screen, but it's dropping off here


![uploads/c5da6c37-bb1e-4218-996a-75297548aa93/Screen Shot 2015-10-03 at 09.12.54.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2Fc5da6c37-bb1e-4218-996a-75297548aa93%2FScreen+Shot+2015-10-03+at+09.12.54.png)

**Comment by IronRunes** [comment]
copy done, images done, footer is not supposed to show up here.
